runtime/Go/antlr/atn_config.go

```diff
@@ -57,35 +56,56 @@ func NewBaseATNConfig7(old *BaseATNConfig) *BaseATNConfig { // TODO: Dup
 	}
 }
 
-func NewBaseATNConfig6(state ATNState, alt int, context PredictionContext) *BaseATNConfig {
-	return NewBaseATNConfig5(state, alt, context, SemanticContextNone)
+// BaseAtnConfigDefaultContext creates a new instance of BaseATNConfig.
+func BaseAtnConfigDefaultContext(state ATNState, alt int, context PredictionContext) *BaseATNConfig {
+	return BaseATNConfigContext(state, alt, context, SemanticContextNone)
 }
 
-func NewBaseATNConfig5(state ATNState, alt int, context PredictionContext, semanticContext SemanticContext) *BaseATNConfig {
+// BaseATNConfigContext creates a new instance of BaseATNConfig.
+func BaseATNConfigContext(state ATNState, alt int, context PredictionContext, semanticContext SemanticContext) *BaseATNConfig {
 	if semanticContext == nil {
 		panic("semanticContext cannot be nil") // TODO: Necessary?
 	}
 
-	return &BaseATNConfig{state: state, alt: alt, context: context, semanticContext: semanticContext}
+	return &BaseATNConfig{
+		state:           state,
+		alt:             alt,
+		context:         context,
+		semanticContext: semanticContext,
+	}
 }
 
-func NewBaseATNConfig4(c ATNConfig, state ATNState) *BaseATNConfig {
+// BaseATNConfigState creates a new instance of BaseATNConfig.
+func BaseATNConfigState(c ATNConfig, state ATNState) *BaseATNConfig {
 	return NewBaseATNConfig(c, state, c.GetContext(), c.GetSemanticContext())
 }
 
-func NewBaseATNConfig3(c ATNConfig, state ATNState, semanticContext SemanticContext) *BaseATNConfig {
+// ATNConfigStateContext creates a new instance of BaseATNConfig.
+func ATNConfigStateContext(c ATNConfig, state ATNState, semanticContext SemanticContext) *BaseATNConfig {
 	return NewBaseATNConfig(c, state, c.GetContext(), semanticContext)
 }
 
-func NewBaseATNConfig2(c ATNConfig, semanticContext SemanticContext) *BaseATNConfig {
+// ATNConfigWithContext creates a new instance of BaseATNConfig.
+func ATNConfigWithContext(c ATNConfig, semanticContext SemanticContext) *BaseATNConfig {
 	return NewBaseATNConfig(c, c.GetState(), c.GetContext(), semanticContext)
 }
 
-func NewBaseATNConfig1(c ATNConfig, state ATNState, context PredictionContext) *BaseATNConfig {
+// ATNConfigWithStateContext creates a new instance of BaseATNConfig.
```

Comment:
I wonder if these comments actually add anything. I don't know go very well at all but it seems like these comments are not adding very much to constructor functions.

Reply:
Actually, they look like useless comments because they don't give additional information, don't explain how an algorithm works, etc.